Formula/safe.rb

```diff
@@ -0,0 +1,40 @@
+class Safe < Formula
+  desc "Vault CLI"
+  homepage "https://github.com/starkandwayne/safe"
+  url "https://github.com/starkandwayne/safe.git",
```

Comment:
Could you use a tarball + `sha256` from the [releases page](https://github.com/starkandwayne/safe/releases) here?
We prefer downloading an archive over doing a `git clone` when possible.

It also looks like there's a new version available: https://github.com/starkandwayne/safe/archive/v0.8.0.tar.gz